sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -476,7 +482,7 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
           // Kill the VM on second ctrl+c
           if (!initialRequest) {
             console.printInfo("Exiting the JVM")
-            System.exit(127)
+            SparkSQLCLIDriver.exit(127)
```

Comment:
Although not introduced by this pr, but I think we should give `127` a definite name.

Reply:
> Although not introduced by this pr, but I think we should give `127` a definite name.

I will raise a new pr for all CLI's exit code.